python/delta/tables.py

```diff
@@ -581,6 +581,21 @@ def upgradeTableProtocol(self, readerVersion: int, writerVersion: int) -> None:
                              type(writerVersion))
         jdt.upgradeTableProtocol(readerVersion, writerVersion)
 
+    def addFeatureSupport(self, feature: str) -> None:
```

Comment:
```suggestion
    def addFeatureSupport(self, featureName: str) -> None:
```

Similar to the case of forName ([Scala](https://github.com/delta-io/delta/blob/8442bc60da5ac45d13e76fb0cd59aacfc0c447ad/spark/src/main/scala/io/delta/tables/DeltaTable.scala#L1033), [Python](https://github.com/delta-io/delta/blob/master/python/delta/tables.py#L393)), we should have the argument's names to be identical as the [Scala counterpart](https://github.com/delta-io/delta/blob/8442bc60da5ac45d13e76fb0cd59aacfc0c447ad/spark/src/main/scala/io/delta/tables/DeltaTable.scala#L555)